pkg/operator/operator.go

```diff
@@ -184,6 +196,11 @@ func New(
 		nodeInformer.Informer(),
 		dnsInformer.Informer(),
 		maoSecretInformer.Informer(),
+		imgInformer.Informer(),
```

Comment:
I think this would technically conflict with https://github.com/openshift/machine-config-operator/pull/3937. Did we want to merge in the same order we did for master?

Reply:
agreed yeah, we can hold off until #3937 merges. We have the time (: